src/backend/distributed/utils/metadata_cache.c

```diff
@@ -34,6 +34,19 @@
 #include "utils/relmapper.h"
 #include "utils/syscache.h"
 
+#include "access/xact.h"
+
+
+/* state which should be cleared upon DROP EXTENSION */
```

Comment:
We should add a note that these shouldn't be accessed directly. Instead, the relevant functions should be called.